drivers/bridge/setup_ipv6.go

```diff
@@ -117,3 +120,31 @@ func setupIPv6Forwarding(config *networkConfiguration, i *bridgeInterface) error
 
 	return nil
 }
+
+func setupNDPProxying(config *networkConfiguration, i *bridgeInterface) error {
+	// Get current NDP default proxying setup
+	ndpProxyDataDefault, err := ioutil.ReadFile(ndpProxyConfDefault)
+	if err != nil {
+		return fmt.Errorf("Cannot read NDP default proxying setup: %v", err)
+	}
+	// Enable NDP default proxying only if it is not already enabled
+	if ndpProxyDataDefault[0] != '1' {
+		if err := ioutil.WriteFile(ndpProxyConfDefault, []byte{'1', '\n'}, ndpProxyConfPerm); err != nil {
+			logrus.Warnf("Unable to enable NDP default proxying: %v", err)
+		}
+	}
+
+	// Get current NDP all proxying setup
+	ndpProxyDataAll, err := ioutil.ReadFile(ndpProxyConfAll)
+	if err != nil {
+		return fmt.Errorf("Cannot read NDP all proxying setup: %v", err)
+	}
+	// Enable NDP all proxying only if it is not already enabled
+	if ndpProxyDataAll[0] != '1' {
```

Comment:
Thanks @mostolog for taking over this.
This is actually the most important comment. Would it work if we enable the ndp proxy only on the host interface specified in the driver option, instead of doing it for `all` the host interfaces ?